paasta_tools/kubernetes_tools.py

```diff
@@ -4155,12 +4153,8 @@ def mode_to_int(mode: Optional[Union[str, int]]) -> Optional[int]:
 
 def update_crds(
     kube_client: KubeClient,
-    desired_crds: Collection[
-        Union[V1CustomResourceDefinition, V1beta1CustomResourceDefinition]
-    ],
-    existing_crds: Union[
-        V1CustomResourceDefinitionList, V1beta1CustomResourceDefinitionList
-    ],
+    desired_crds: Collection[Union[V1CustomResourceDefinition]],
+    existing_crds: Union[V1CustomResourceDefinitionList],
```

Comment:
```suggestion
    desired_crds: Collection[V1CustomResourceDefinition],
    existing_crds: V1CustomResourceDefinitionList,
```
we don't need a union anymore since we're using a single type now :)